docs/user-guide/recipes.rst

```diff
@@ -1083,7 +1083,7 @@ utility. Let's create and sync a repo::
 
 Now let's build a package langpacks entry for the hyphen package::
 
-   $ pulp-admin rpm repo uploads langpacks -i hyphen -n hyphen-%s --repo-id repo_1
+   $ pulp-admin rpm repo uploads langpacks --name hyphen --install hyphen-%s --repo-id repo_1
```

Comment:
Using full option names can feel pedantic. But given that documentation is designed to educate, I figure this change is beneficial.